internal/network/manifests_generator.go

```diff
@@ -55,7 +55,7 @@ kind: MachineConfig
 metadata:
   labels:
     machineconfiguration.openshift.io/role: {{.ROLE}}
-  name: {{.ROLE}}s-chrony-configuration
+  name: 99-{{.ROLE}}-chrony-configuration
```

Comment:
According to https://docs.openshift.com/container-platform/4.5/installing/install_config/installing-customizing.html the naming of the role should be plural. Please keep the 's' at the end

Reply:
OK